docker/README.md

```diff
@@ -0,0 +1,33 @@
+# Benchmarks in Docker
```

Comment:
Doesn't need to be in this PR, but have you considered adding a docker-compose.yml file to make it easier to start combinations of containers?